meterfs/file_phx.c

```diff
@@ -30,11 +31,19 @@ static inline void file_prepareDevCtl(msg_t* msg)
 }
 
 
+static int lookup_rel(const char *name, oid_t *file, oid_t *dev)
+{
+	char buffer[64];
+	(void)snprintf(buffer, sizeof(buffer), "%s%s", pathPrefix, name);
```

Comment:
```suggestion
	if (snprintf(buffer, sizeof(buffer), "%s%s", pathPrefix, name) >= sizeof(buffer)) {
		return -ENAMETOOLONG;
```

Reply:
Also, do `"%s/%s"` just to be sure (path might be specified without trailing `/`)